version.sbt

```
@@ -1,2 +1,2 @@
-version in ThisBuild := "2.0.0-RC3"
+version in ThisBuild := "2.0.0"
```

```suggestion
version in ThisBuild := "2.0.0-RC4"
```

Because of the zio dependency only. Your call

---

`monix` and `zio` are only pulled by the `examples` which are not published as a dep.